keybase_proofs/views.py

```diff
@@ -59,6 +56,35 @@ def verify_proof(user, sig_hash, kb_username):
         return False
 
 
+def is_proof_live(user, sig_hash, kb_username):
+    """
+    Checks the proof status in Keybase via the `sig/proof_live` endpoint.
+    Returns a tuple (`proof_valid`, `proof_live`) indicating if the signature
+    is live and if the proof is publicly verifiable to the Keybase
+    client/server, respectively.
+
+    Before storing a signature `proof_valid=True` must hold. Services can
+    optionally check the status of signatures periodically and verify
+    `proof_live=True` as well.
```

Comment:
it looks like this isn't being called by anything right now. which makes sense. i think a lot of people are going to get this wrong. i fear mostly people checking that the proof is live before persisting it. or just generally misinterpreting the difference between valid and live. maybe we can add a little to this comment. 
```
Keybase suggests an asynchronous task that calls this endpoint during
the proof creation flow. It will return (valid=true, live=false) until
Keybase has seen it being served from this service's API, at which
point it will update to (valid=true, live=true). This is the happy path. 
It may also make sense to call this endpoint periodically
or whenever a user inspects their own proof/profile/settings to update local records. 
If a user revokes their proof from Keybase, this will return (valid=false, live=false).
```
maybe too much. /shrug

Reply:
i'll work it in, thanks!